README.md

```diff
@@ -11,6 +11,7 @@ The `efs-utils` package has been verified against the following Linux distributi
 | Amazon Linux 2017.09 | `rpm` | `upstart` |
 | Amazon Linux 2 | `rpm` | `systemd` |
 | CentOS 7 | `rpm` | `systemd` |
+| CentOS 8 | `rpm` | `systemd` |
```

Comment:
Please do not change this file currently, we have a full process of qualifying linux distributions.

Reply:
README reverted in 6fa35aa